src/current/cockroachcloud/alerts-page.md

```diff
@@ -31,24 +32,30 @@ If alerts are enabled, {{ site.data.products.db }} will send alerts to [specifie
 **CMEK:**
 
 - Cluster node unable to start due to CMEK key access failure.
-- Encrypted backup failed due to CMEK key access failure.
+{% comment %}- Encrypted backup failed due to CMEK key access failure.{% endcomment %}
 
 If you receive an alert repeatedly, you may need to [optimize your workload](../stable/make-queries-fast.html) or [scale your {{ site.data.products.db }} cluster](cluster-management.html?filters=dedicated#add-or-remove-nodes-from-a-cluster).
 
-[Org Administrators](authorization.html#org-administrator-legacy) will also receive email alerts when your cluster undergoes an automatic [patch version upgrade](upgrade-policy.html#patch-version-upgrades).
+[Org Administrators](authorization.html#org-administrator-legacy) also receive email alerts when your cluster undergoes an automatic [patch version upgrade](upgrade-policy.html#patch-version-upgrades).
 
 {{site.data.alerts.callout_success}}
-When scaling your cluster, we recommend first scaling VMs to include more than 2 vCPUs each. If this doesn't sufficiently improve performance, then add more nodes.
+When scaling your cluster, Cockroach Labs recommends that VMs for production clusters include a minimum of {% include {{ page.page_version }}/prod-deployment/provision-cpu.md threshold='minimum' %}. If increasing vCPU doesn't sufficiently improve performance, then add more nodes. Refer to [Recommended Production Settings](/docs/{{site.versions["stable"]}}/recommended-production-settings.html).
```

Comment:
I think we should keep this update out of this PR, and I would like it to be a separate one which can be reviewed by relevant people.

Reply:
Discussed offline, this info doesn't belong in this page anyway. I'll remove it.